Tests/SwiftParserTest/translated/StringLiteralEofTests.swift

```diff
@@ -105,7 +105,7 @@ final class StringLiteralEofTests: XCTestCase {
             foo
             \(<#expression#>)"""
         """##
-      // FIXME: The closing delimiter should be put on the new line
+        // FIXME: The closing delimiter should be put on the new line
```

Comment:
I think your local build of swift-format is too old. The formatting of this line changed with https://github.com/apple/swift-format/pull/480.

Reply:
Ah, that makes sense.